docs/tutorial/part_1.rst

```diff
@@ -188,25 +188,25 @@ Your app directory should now look like::
 In the ``chat/urls.py`` file include the following code::
 
     # chat/urls.py
-    from django.conf.urls import url
+    from django.urls import path
     
     from . import views
     
     urlpatterns = [
-        url(r'^$', views.index, name='index'),
+        path('chat',views.index, name='index'),
```

Comment:
It seems this url is swapped with [that one](https://github.com/django/channels/pull/1238/files#diff-52ab88a5732297f928e9c2f3d5698541R208), can you confirm please ?

Reply:
I did not swap it, I adjusted it a bit, for example, if the user types http://127.0.0.1:8000 then it will take him to the chat app, and then with the pattern `chat/`, it'll take them to views.index, I did that because in the future if we want to add a new pattern, we'll not always start with `chat`.it gives us more flexibility in our url patterns

Reply:
Sorry to be a pain, but please move the `chat` prefix back to the `include()` call below. 

That way, I can leave my app untouched by include it with a different prefix. I can also include multiple instances of the app under different prefixes. If you change it we need to open the editor to change prefix, and all includes will need an extra path fragment to avoid conflicts.